packages/hadron-build/lib/generate-package-lock.js

```diff
@@ -1,8 +1,21 @@
 const path = require('path');
-const { promises: fs } = require('fs');
 const { Arborist, Shrinkwrap } = require('@npmcli/arborist');
 const pacote = require('pacote');
-const { withProgress } = require('./monorepo/with-progress');
+const ora = require('ora');
+
+async function withProgress(text, fn, ...args) {
+  const spinner = ora(text).start();
+  try {
+    const result = await fn.call(spinner, ...args);
+    spinner.succeed();
+    return result;
+  } catch (e) {
+    if (spinner.isSpinning) {
+      spinner.fail();
+    }
+    throw e;
+  }
+}
```

Comment:
Now that this script is part of the build lib I think we can just drop all those progress spinners tbh, no need to move them over also (maybe replace then with debug calls to be consistent with other parts of hadron-build)

Reply:
sounds good